Prox/Prox/Widgets/PlaceReviewView.swift

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import QuartzCore
+
+class PlaceReviewView: UIView {
+
+    var color: UIColor? {
+        didSet {
+            self.reviewScore.color = color
+        }
+    }
+
+    var score: Int = 0 {
+        didSet {
+            reviewScore.score = score
+        }
+    }
+
+    lazy var reviewSiteLogo: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.borderColor = UIColor.yellow.cgColor
+        imageView.layer.borderWidth = 1.0
+        return imageView
+    }()
+
+    lazy var reviewScore: ReviewScoreView = {
+        let view = ReviewScoreView()
+        view.color = self.color
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var numberOfReviewersLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Yeah, that is probably a good idea.

This method of using AutoLayout only came about in iOS9, but because on FxiOS we were supporting iOS8 we couldn't use it. Therefore I haven't yet come up with any 'tricks' to solve this problem. It is annoying as f*** to have to remember to add `translatesAutoresizingMaskIntoConstraints` to every view.